Exception thrown when using clear function of JsonPropertiesEditor

When using the public clear function of the JsonPropertiesEditor the following Exception is thrown:

> Exception in Application start method
Is valid: false
Exception in thread "main" java.lang.RuntimeException: Exception in Application start method
	at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:917)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$1(LauncherImpl.java:182)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.RuntimeException: A bound value cannot be set.
	at javafx.beans.property.BooleanPropertyBase.set(BooleanPropertyBase.java:140)
	at com.github.hanseter.json.editor.JsonPropertiesEditor.rebindValidProperty(JsonPropertiesEditor.kt:107)
	at com.github.hanseter.json.editor.JsonPropertiesEditor.clear(JsonPropertiesEditor.kt:100)
	at com.github.hanseter.json.editor.JsonPropertiesEditorTestApp.start(JsonPropertiesEditorTestApp.kt:33)
	at com.sun.javafx.application.LauncherImpl.lambda$launchApplication1$8(LauncherImpl.java:863)
	at com.sun.javafx.application.PlatformImpl.lambda$runAndWait$7(PlatformImpl.java:326)
	at com.sun.javafx.application.PlatformImpl.lambda$null$5(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$6(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
	at com.sun.glass.ui.win.WinApplication.lambda$null$3(WinApplication.java:177)
	... 1 more

--- a/src/main/kotlin/com/github/hanseter/json/editor/JsonPropertiesEditor.kt
+++ b/src/main/kotlin/com/github/hanseter/json/editor/JsonPropertiesEditor.kt
@@ -103,6 +103,7 @@ class JsonPropertiesEditor(
 
 	private fun rebindValidProperty() {
 		if (idsToPanes.isEmpty()) {
+			valid_.unbind()
 			valid_.set(true)
 		} else {
 			valid_.bind(idsToPanes.values.map { it.valid as ObservableBooleanValue }.reduce { a, b -> Bindings.and(a, b) })

